### ISSUE TITLE
Fuel Gauges not showing and planes not taking off

### ISSUE THREAD

**Issue by TheTrueHonker** [jaihysc/Factorio-AircraftRealism#7]
After long testing and looking at the source code I can't find the issue.

The fuel gauges at the top left corner are not showing up and even since I'm on a runway and way over the takeoff-speed-limit, the plane won't take off.

The strict-runway-setting is set to false.

**Comment by jaihysc** [comment]
Hi, thanks for posting an issue here,

I just tested this on the latest Factorio release (0.18.9) and everything appears to be working.

Can you tell me the following:
- Your game version
- Your active mods

The gauges should appear any time you enter an aircraft, can you ensure the following:
- `Mod settings > Per player > Aircraft Realism > Enable gauges` is checked

This is probably caused by another mod, does the issue still persist even with all other mods disabled?

**Comment by TheTrueHonker** [comment]
A friend of mine tried it again, but with a "different" plane, dunno what he means.

Apperantly it's working now. The "bug" was caused by the AAI mod. The plane I tested it with was the AAI-Mod-Plane with the same stats, name and skin as the AircraftRealism-Mod.

TLDR: Used the wrong plane (AAI-Mod); it's working now
